mlflow/server/js/src/setupProxy.js

```diff
@@ -0,0 +1,8 @@
+const proxy = require('http-proxy-middleware');
+
+function setupProxyFn(app) {
+  app.use(proxy('/ajax-api', { target: 'http://localhost:5000' }));
+  app.use(proxy('/get-artifact', { target: 'http://localhost:5000', ws: true }));
+}
+
+module.exports = setupProxyFn;
```

Comment:
Where is this used?

Reply:
It was set in `package.json` before. It is for development purposes.
But in react-script v2 or higher for "complexe" proxy setup, you have to use `setupProxy.js` : https://create-react-app.dev/docs/proxying-api-requests-in-development#configuring-the-proxy-manually 

> Note: You do not need to import this file anywhere. It is automatically registered when you start the development server.